### ISSUE TITLE
Excessive formatting in log.info if called with one argument

### ISSUE THREAD

**Issue by fsitedev** [tarantool/tarantool#1773]
```
unix/:/var/run/tarantool/mediator-queue.control> require('log').info("%20")
---
- error: 'builtin/log.lua:38: invalid option ''%20'' to ''format'''
```
Tarantool 1.6.8-761-g8b23c8d

**Comment by rtsisyk** [comment]
Hi, we use Lua's [string.format()](https://www.lua.org/manual/5.1/manual.html#pdf-string.format) under the hood.

**Comment by fsitedev** [comment]
Yes, it's quite clear but maybe it will be safer if log will not do any formatting if only one argument is passed into it.
I've faced with situation when simple `log.info(var)` call leads to application crash just because 'var' occasionally contains '%' symbol.

**Comment by Totktonada** [comment]
My 2 coins: logging should never raise an error.

A logger should do the best to write given message in some form.
